packages/fx-core/resource/package.nls.json

```diff
@@ -399,7 +399,7 @@
   "core.common.UrlProtocolNotSupported": "Server url is not correct: protocol %s is not supported, you should use https protocol instead.",
   "core.common.RelativeServerUrlNotSupported": "Server url is not correct: relative server url is not supported.",
   "core.common.ResolveServerUrlFailed": "Unable to resolve the server URL: please make sure that the environment variable %s is defined.",
-  "core.common.OperationOnlyContainsOptionalParam": "Operation %s only contains optional parameters which are not supported",
+  "core.common.OperationOnlyContainsOptionalParam": "Operation %s only contains multiple optional parameters, so we use the first optional parameter for the command.",
```

Comment:
remove "only"?

Reply:
@sffamily , do you have any ideas about this message, I have left a message in your Teams about this~

Reply:
Operation %s contains multiple optional parameters. We use the first optional parameter for this command.

Reply:
Sorry, use this version:

Operation %s contains multiple optional parameters. The first optional parameter is used for this command.

Reply:
Thanks, updated